tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot

```diff
@@ -6,16 +6,18 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
 
 *** Test Cases ***
 Simple docker volume create
-    Log  Not really implemented  WARN
-    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --opt VolumeStore=datastore1 --opt Capacity=1
-    #Should Be Equal As Integers  ${rc}  1
+    Log  Not supported yet  WARN
+    # Auto-named volumes not supported yet
+    #${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --opt VolumeStore=default --opt Capacity=1
+    #Should Be Equal As Integers  ${rc}  0
 
 Docker volume create named volume
-    Log  Not really implemented  WARN
-    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test --opt VolumeStore=datastore1 --opt Capacity=1
-    #Should Be Equal As Integers  ${rc}  1
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test --opt VolumeStore=default --opt Capacity=1
```

Comment:
This might need to be a little bigger. 1MB is somewhat small for a disk. not sure if it really matters for the test though. I don't know how big the test environment disks are. I imagine 256 or 512 might be a bit safer.